### PR TITLE
[Docs] `default-props-match-prop-types`, `require-default-props`, `sort-prop-types`: fix typos

### PR DESCRIPTION
**Pull request by nix6839**

### CODE REVIEW

File: docs/rules/require-default-props.md [hunk 1]
```diff
@@ -165,7 +165,7 @@ MyStatelessComponent.propTypes = {
 };
 
 MyStatelessComponent.defaultProps = {
-    bar: 'some default'
```
Comment (ljharb) [inline docs/rules/require-default-props.md:168]: let's also fix the indentation in the previous example

**Review by ljharb — CHANGES_REQUESTED** [review]
thanks!

File: docs/rules/require-default-props.md (hunk 1 above)
Comment (nix6839) [reply docs/rules/require-default-props.md:168]: Where is typo in the previouse example? I don't find that. It's means another document?

Comment (ljharb) [reply docs/rules/require-default-props.md:168]: the previous example seems to have one-space indentation

Comment (nix6839) [reply docs/rules/require-default-props.md:168]: Sorry, I don't understand. Are you talking about revert?

Comment (ljharb) [reply docs/rules/require-default-props.md:168]: I'm talking about line 165 - it looks like the `};` is an extra space indented. is that just github's UI?

Comment (nix6839) [reply docs/rules/require-default-props.md:168]: There is no extra space indented. Just github's UI :)

**Review by ljharb — APPROVED** [review]

**Pull request merged**